package.json

```diff
@@ -31,7 +39,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "typescript": "^4.2.4"
```

Comment:
Please list typescript to a dev dependency. We don't need to install typescript as a regular dependency to get the type definitions for installs.

Reply:
Change done